### DISCUSSION TITLE
Compile and view docs

### DISCUSSION THREAD

**Discussion by alexachso** [Live-Charts/LiveCharts2#412] [Q&A] [answered]
Hi, 

Can someone guide me on how to compile and view all the .md files in docs as they intended? What I mean is, I have Visual Studio Code and I can open the .md files individually and preview its contents, but all the references to gifs and pngs that are stored in the folder "_assets" are not rendered.

Is there any particular VSCode extension that I can use to compile the docs folder as a beautiful HTML to render all the images from _assets folder linked into the .md files?

Thank you!

**Comment by beto-rodriguez** [comment, marked as answer]
For now the docs can only be compiled with my private repo, now the library seems stable enough to publish the docs, I will publish a new site with the compiled docs asap.

**Reply by alexachso** [reply]
Thank you for taking the time to answer :)

**Reply by BarkenBark** [reply]
Hi! Sorry for bumping an old thread, but I was just wondering if your private repo for compiling the docs is based on/uses some popular tool/engine? I really liked the way you've structured the docs source in this repo, and am looking to achieve something similar in my work.

Cheers!

**Reply by beto-rodriguez** [reply]
Yes, it uses scriban, you can find more info about the engine here: https://github.com/beto-rodriguez/LiveCharts2/issues/1024

**Reply by BarkenBark** [reply]
Thank you very much!
